themes/tiddlywiki/vanilla/base.tid

```diff
@@ -98,6 +98,13 @@ html button {
 	border-color: <<colour button-border>>;
 }
 
+html button:disabled {
+	color: <<colour button-foreground>>80;
```

Comment:
I think this technique is brittle because it only works with six digit hex colours. While we try to enforce that restriction within the core palettes, there's nothing to stop palettes from using the CSS `rgb()` function to specify colours.

Reply:
I think we should not use opacity at all. It does not work well, if users apply custom styles. They are not used to deal with opacity in the default settings.